run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -211,6 +211,21 @@ RRTMG_Timestep_sec=10800
 # Mass tuning factor is used in the HEMCO DustDead extenstion for GEOS-Chem
 # benchmarking and is resolution-dependent. We recommend using offline dust
 # instead of the online extension for GCHP science runs.
+#
+# NOTE: We have retuned the dust scaling factors to account for the
+# fact that the HEMCO DustDead extension uses USTAR directly from the
+# meteorology instead of computing it from U10M and V10M.  For now we
+# have computed new scale factors for C24 to allow benchmarks to proceed.
+# Other resolution/met field combinations will be added later.
+#   -- Bob Yantosca (05 Feb 2025)
+
+metField=$(grep "met_field:" geoschem_config.yml)
```

Comment:
Why isn't this section within the check on whether dustdead is on?

Reply:
@lizziel: We have simplified the logic where mass tuning factors are set in commit d08e541762be53774d818deaf691a59df4890c47.